tests/api_test.py

```diff
@@ -219,6 +255,74 @@ def test_get_newdle(flask_client, dummy_newdle):
     }
 
 
+@pytest.mark.usefixtures('db_session')
+def test_update_invalid_newdle(flask_client, dummy_uid):
+    resp = flask_client.patch(
+        url_for('api.update_newdle', code='xxx'),
+        json={'title': 'foo'},
+        **make_test_auth(dummy_uid),
+    )
+    assert resp.status_code == 404
+    assert resp.json == {'error': 'Specified newdle does not exist'}
+
+
+@pytest.mark.usefixtures('db_session')
+def test_update_newdle_unauthorized(flask_client, dummy_newdle):
+    resp = flask_client.patch(
+        url_for('api.update_newdle', code='dummy'),
+        json={'title': 'foo'},
+        **make_test_auth('someone'),
+    )
+    assert resp.status_code == Forbidden.code
```

Comment:
I'd just hardcode `403` here since it's common knowledge that 403 is the http status code for Forbidden